src/components/forms/phone-input.jsx

```diff
@@ -53,9 +53,9 @@ var PhoneInput = React.createClass({
                                      label={null}
                                      disabled={this.isFormDisabled() || this.props.disabled}
                     />
+                    {this.renderErrorMessage()}
                 </div>
                 {this.renderHelp()}
```

Comment:
I was wrong about this being outside the row on the others: https://github.com/twisty/formsy-react-components/blob/master/src/input.js#L92 https://github.com/twisty/formsy-react-components/blob/master/src/textarea.js#L59